### PR TITLE
Enhancement done to support tenant wise and date range push data.

### PR DESCRIPTION
**Pull request by manastanmay-eGov**
Enhancement done in Tl service, Collection Service and Property Services

### CODE REVIEW

File: business-services/collection-services/src/main/java/org/egov/collection/repository/PaymentRepository.java [hunk 1]
```diff
@@ -248,11 +248,53 @@ public void updateFileStoreId(List<Map<String,String>> idToFileStoreIdMaps){
...
+        boolean whereCluaseApplied= false ;
+        boolean isTenantPresent= true ;
+
         Map<String, Object> preparedStatementValues = new HashMap<>();
+
         preparedStatementValues.put("offset", paymentSearchCriteria.getOffset());
         preparedStatementValues.put("limit", paymentSearchCriteria.getLimit());
 
-        return namedParameterJdbcTemplate.query("SELECT id from egcl_payment ORDER BY createdtime offset " + ":offset " + "limit :limit", preparedStatementValues, new SingleColumnRowMapper<>(String.class));
+        if(paymentSearchCriteria.getTenantId() != null && !paymentSearchCriteria.getTenantId().equals("uk")) {
```
Comment (vinothrallapalli-eGov) [inline business-services/collection-services/src/main/java/org/egov/collection/repository/PaymentRepository.java:261]: Why the tenant id (UK) is hard-coded?

File: municipal-services/tl-services/src/main/java/org/egov/tl/repository/TLRepository.java [hunk 2]
```diff
@@ -139,15 +140,31 @@ public List<TradeLicense> getPlainLicenseSearch(TradeLicenseSearchCriteria crite
...
 		preparedStmtList.add(criteria.getLimit());
 
-		return jdbcTemplate.query("SELECT id from eg_tl_tradelicense ORDER BY createdtime offset " +
-						" ? " +
-						"limit ? ",
-				preparedStmtList.toArray(),
-				new SingleColumnRowMapper<>(String.class));
-	}
+        if(!StringUtils.isEmpty(criteria.getTenantId())) {
+            System.out.println(criteria.getTenantId());
```
Comment (vinothrallapalli-eGov) [inline municipal-services/tl-services/src/main/java/org/egov/tl/repository/TLRepository.java]: Remove sysout

**Review by vinothrallapalli-eGov — CHANGES_REQUESTED** [review]

File: business-services/collection-services/src/main/java/org/egov/collection/repository/PaymentRepository.java (hunk 1 above)
Comment (manastanmay-eGov) [reply business-services/collection-services/src/main/java/org/egov/collection/repository/PaymentRepository.java:261]: As this repo abd branch is  dedicated to UKD so have specified it as is . But if required then can make this code base generic as well. 
But i have referred Punjab code changes for the same and in there the state pb is hard coded.(https://github.com/egovernments/business-services/commit/73f419c7e5a63afb58fa43a67a54267a330e4819)

File: municipal-services/tl-services/src/main/java/org/egov/tl/repository/TLRepository.java (hunk 2 above)
Comment (manastanmay-eGov) [reply municipal-services/tl-services/src/main/java/org/egov/tl/repository/TLRepository.java]: Removed

**Review by vinothrallapalli-eGov — APPROVED** [review]

**Pull request merged**